.github/workflows/release-docker.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Publish Latest to Docker
         uses: docker/build-push-action@v1.1.0
         # only on releases and latest Python and PyTorch
-        if: matrix.python_version == "3.9" && matrix.pytorch_version == "1.10"
+        if: matrix.python_version == '3.9' && matrix.pytorch_version == '1.10'
```

Comment:
Using this PR to fix this error:

https://github.com/PyTorchLightning/pytorch-lightning/actions/runs/1829634458/workflow#L42